### ISSUE TITLE
failure on dc_mlx5/uct_p2p_err_test.remote_access_error/1

### ISSUE THREAD

**Issue by alinask** [openucx/ucx#1834]
```
14:22:35 [----------] 4 tests from dc_mlx5/uct_p2p_err_test
14:22:35 [ RUN      ] dc_mlx5/uct_p2p_err_test.remote_access_error/1
14:22:35 [     INFO ] < Error on QP 0xdb87 wqe[0]: Remote access (synd 0x13 vend 0x88) opcode RDMA_WRITE >
14:22:35 [     INFO ] < Error Endpoint timeout was not handled for ep 0x2f47e60 >
14:22:36 [hpc-test-node3:65395:0] Caught signal 6 (Aborted: tkill(2) or tgkill(2))
16:44:23 Build timed out (after 150 minutes). Marking the build as failed.
16:44:23 Build was aborted
```

http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/4555/label=hpc-test-node3,worker=2/console

**Comment by hoopoepg** [comment]
this issue is relevant for OFED 4.2:
```
[13:27:01;ucx/test/gtest]$ ./gtest --gtest_filter=dc_mlx5/uct_p2p_err_test.remote_access_error/0
[     INFO ] checking dc: yes
[     INFO ] checking dcx: yes
[     INFO ] checking ud: yes
[     INFO ] checking udx: yes
[     INFO ] checking udrc: yes
[     INFO ] checking cmrcx: yes
[     INFO ] checking shm: yes
[     INFO ] checking udrcx: yes
[     INFO ] checking ugni: no
[     INFO ] checking self: yes
[     INFO ] checking tcp: yes
[     INFO ] checking all: yes
[     INFO ] checking rc: yes
[     INFO ] checking rc_x: yes
[     INFO ] checking ud_mlx5: yes
[     INFO ] Using random seed of 6033
Note: Google Test filter = dc_mlx5/uct_p2p_err_test.remote_access_error/0
[==========] Running 1 test from 1 test case.
[----------] Global test environment set-up.
[----------] 1 test from dc_mlx5/uct_p2p_err_test
[ RUN      ] dc_mlx5/uct_p2p_err_test.remote_access_error/0
[     INFO ] < IB Async event on mlx5_0: DCT access error on DCTN UNKNOWN >
[vegas09:4633 :0] Caught signal 11 (Segmentation fault: address not mapped to object at address 0x38)
[     INFO ] < Error on QP 0x8c49 wqe[0]: Remote access (synd 0x13 vend 0x88) opcode RDMA_WRITE >
[     INFO ] < Error Endpoint timeout was not handled for ep 0x30b9030 >
==== backtrace ====
 0 0x0000000000009bd0 __GI___pthread_mutex_destroy()  :0
 1 0x000000000000a9bf ibv_ack_async_event()  ???:0
 2 0x00000000000255e2 uct_ib_async_event_handler()  /labhome/sergeyo/ucx/src/uct/ib/base/ib_device.c:178
 3 0x000000000003b758 ucs_async_handler_dispatch()  /labhome/sergeyo/ucx/src/ucs/async/async.c:199
 4 0x000000000003ba22 ucs_async_dispatch_handlers()  /labhome/sergeyo/ucx/src/ucs/async/async.c:232
 5 0x000000000003f050 ucs_async_thread_func()  /labhome/sergeyo/ucx/src/ucs/async/thread.c:113
 6 0x0000000000007dc5 start_thread()  pthread_create.c:0
 7 0x00000000000f6ced __clone()  ???:0
===================
[vegas09:4633 :0] Process frozen, press Enter to attach a debugger...
```

```
$ ofed_info -s
MLNX_OFED_LINUX-4.2-0.2.2.0:
```

**Comment by yosefe** [comment]
fixed in #4e773a8f58f800b7265b19254673d1256f92bcb4
